### PR TITLE
Feature/28 add checkbox for flexible working arrangements available next to working pattern

### PR DESCRIPTION
**Pull request by despo**
this PR also introduces a simple_form wrapper that can be used across all checkbox components with
`wrapper: :inline_checkbox`

### CODE REVIEW

**Comment by despo** [comment]
[add checkbox for 'flexible working arrangements available' next to working pattern](https://trello.com/c/0L8PGDAp/28-add-checkbox-for-flexible-working-arrangements-available-next-to-working-pattern)

**Comment by tahb** [comment]
Doesn't seem to work properly in Firefox.
screen shot 2018-04-09 at 17 27 11

**Comment by despo** [comment]
@tahb  On my FF...
 check-box

File: config/initializers/simple_form.rb [hunk 1]
```diff
@@ -62,7 +62,7 @@
   # Defaults to :nested for bootstrap config.
   #   inline: input + label
   #   nested: label > input
-  config.boolean_style = :nested
+  config.boolean_style = :inline
```
Comment (despo) [inline config/initializers/simple_form.rb]: @tahb I reverted this on my local, which was causing the checkbox not to appear.

**Comment by tahb** [comment]
I've looked at the job posting standard to see how it's proposed that this information is stored and represented. It's a touch vague but this looks relevant:

> workHours | Text | The typical working hours for this job (e.g. 1st shift, night shift, 8am-5pm).

Perhaps we should break this information into a new form step (which was advised in our alpha assessment) and make it Yes by default as per https://www.gov.uk/flexible-working. We can use this new checkbox to help us set our `weekly_hours` field.

If yes: weekly_hours = `flexible`
If no: weekly_hours = `30`

Good to chat about this in more detail with others to get some feedback!

**Comment by tahb** [comment]
Closing in favour of https://github.com/dxw/teacher-vacancy-service/pull/148

**Comment by tahb** [comment]
Closed the wrong one, whoops.

**Comment by despo** [comment]
flexible-working

**Pull request merged**